ios/RNGeth.swift

```diff
@@ -82,16 +109,201 @@ class RNGeth: RCTEventEmitter, GethNewHeadHandlerProtocol {
             NSLog("@", NSErr)
         }
     }
-    
+
+    /**
+     * List accounts
+     * @return Return All accounts from keystore
+     */
+    @objc(listAccounts:rejecter:)
+    func listAccounts(resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
```

Comment:
imo we shouldn't expose the keystore indices as it just creates confusion for consumers

Reply:
True, we can get rid of that. I felt like keeping the `Account` interface, but we can just return a string array here with addresses.